redux/reducers/userReducer/index.js

```diff
@@ -0,0 +1,10 @@
+const userReducer = (state=[],action) => {
+    switch (action.type) {
+        case 'USER_AUTH':
+            return state;
+        default:
+            return state;
+    }
```

Comment:
why you are returning the same state if `action` is `USER_AUTH`?

Now one more thing, I strongly recommend you don't use string for action. create a variable like `const USER_AUTH = 'USER_AUTH'` and then use that variable and export from `actions` if you want to use them into your `reducer`. This will decrease the error probability to very low and it is a good way.

Reply:
Yeah we will have to write the entire front end and the structure as this was when I just left the project cz of other work.